doc/docs/Python_User_Interface.md

```diff
@@ -87,40 +87,40 @@ control various parameters of the Meep computation.
 
 ```python
 def __init__(self,
-             cell_size,
-             resolution,
-             geometry=None,
-             sources=None,
-             eps_averaging=True,
-             dimensions=3,
-             boundary_layers=None,
-             symmetries=None,
-             force_complex_fields=False,
-             default_material=Medium(),
-             m=0,
-             k_point=False,
+             cell_size: meep.geom.Vector3 = None,
+             resolution: float = None,
```

Comment:
What happens if the user then passes an integer?

Reply:
I modified the type of `resolution` to `Union[float, int]` in order to support both number types.